support-frontend/assets/components/subscriptionCheckouts/address/addressFieldsStore.jsx

```diff
@@ -81,6 +81,9 @@ const checkpostCodeLength = (input: string | null): boolean => ((input == null)
 const isStateNullable = (country: Option<IsoCountry>): boolean =>
   country !== 'AU' && country !== 'US' && country !== 'CA';
 
+const checkStateFieldLength = (state: string | null): boolean =>
```

Comment:
how about this? 
```suggestion
const isStateFieldShortEnough = (state: string | null): boolean =>
```

Reply:
Thanks John! I ended up making this `checkLength` based on @i-hardy's suggestion that a more reusable function would be better.

Reply:
fair enough, we could have followed both suggestions and called it `isFieldShortEnough` but either way it's pretty minor on such a simple function.